torch/optim/lr_scheduler.py

```diff
@@ -59,9 +78,34 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_lr(self):
+    def get_computed_values(self):
+        # Return last computed learning rate by current scheduler
+        try:
+            return self._last_computed_values
+        except AttributeError:
+            raise RuntimeError("Scheduler needs to step to get last computed values.")
```

Comment:
You haven't called `.step()` even once on the Scheduler, so there are no last computed values

Reply:
Indeed, I'm capturing the `AttributeError` and returning a `RuntimeError` telling the user to take a step with scheduler. I've clarified the comment. Is that what you meant?

Reply:
`step` is called in `__init__`.